introduction.md

```diff
@@ -0,0 +1,227 @@
+# Introduction
+
+[Insert text on who this toolkit is for]
+
+[ Do we want to mention goal-setting and strategic approaches here? ]
+
+## A Brief History of Open Source Collaboration
+
+Software has always been shared.  Few significant software development
+efforts have ever been solo acts.  Early programmers tended to be
+quite free with their code.  And in a time when a computer weighed
+more than a car and cost more than a truck, nobody worried too much
+about restricting the instructions necessary to drive them.  Copyright
+existed, but historical evidence suggests that nobody paid much
+attention to it except the lawyers.
+
+All of this early sharing contained the seeds of open source, but the
+specific type of distributed collaboration we recognize today didn't
+quite exist yet.  Sharing and improvements were ad-hoc.  Process was
+idiosyncratic.  We simply didn't have the technology to share software
+and collaboration as fluidly as we do now.  Software wasn't as
+portable.  It was often tied quite closely to the machine on which it
+was written.  We didn't have the internet, websites, bug trackers,
+distributed version control.  By today's standards, software
+collaboration was relatively inefficient and unreliable.
+
+Over time, that began to change.  Much software became portable across
+architectures, which made it more valuable.  More to the point, it
+sometimes made one company's software valuable to its competitors.  To
+manage this situation, companies began to get serious about their
+copyrights.  Suddenly, the software that had been the subject of so
+much free (though clunky) collaboration became more difficult to work
+on because of copyright.
+
+In order to restore that previous collaborative dynamic, some
+developers began to share software under a new type of license that
+encouraged sharing instead of discouraging it.  Later, people wrote
+more licenses in the same vein.  They also developed technology and
+practices that made it easier to collaborate across distance,
+boundaries, and time.
+
+At some point, those collaborative practices and the licenses designed
+for sharing became known as "Free" or "Open Source" and more
+specifically identified as licenses that respect the [Open Source
+Definition](https://opensource.org/osd).  In the decades since,
+software made with these methods has taken over the world.  Every
+major consumer electronics product has at least some open source
```

Comment:
open-source software

Reply:
@lacabra +1 on hyphenating "open-source" when used as an adjective.

Interestingly, although the Open Source Definition does hyphenate it in that circumstance, for some reason that style hasn't really taken hold elsewhere and most people use unhyphenated "open source" even when it's an adjective.  Personally this has always annoyed me, because I agree it would be better hyphenated then, but generally I've just left it unhyphenated because I didn't feel it was worth taking a stand about... However, now that you're *inviting* us to do the thing I always wanted to do anyway, I heartily agree!  :-)

I have made [a note](https://github.com/unicef/publicgoods-toolkit/issues/25) for us to do this across the entire toolkit.  My preference would be to do it as one commit, since conceptually it's really a single change.  This implies doing it after all current PRs have been merged (including PRs that still have comments that we're going to address post-merge, as we discussed in the meeting today).

If you would like the big hyphenate-open-source change to be a PR, we can do that, but my inclination is not to.  Instead, we'd just commit it directly on `main`, as this is a case where the PR overhead would have more cost than benefit.  Let me know if you prefer a PR though; if you do, I'm happy to do one if you strongly prefer it.

One other thing: with a repeatable comment that would apply identically to multiple easily-findable places, there's no need to flag other occurrences after the first one, at least not for our sake.  For example, you can just point out the hyphenation of "open-source" once, in the first place you happen to encounter it, and we'll know to go searching for the rest.  That way we can avoid having lots of small comments that all say essentially the same thing in the PR review, which in turn makes scanning the PR review a bit easier.

Reply:
Thank you, and understood on only flagging on instance, will do